java-checks/src/test/files/checks/IgnoredReturnValueCheckJava13.java

```diff
@@ -0,0 +1,11 @@
+class Foo {
+
+  void bar(String s, String cond) {
+    String v = switch (cond) {
+     case "1" -> "aValue";
+     case "2" -> s.toString();
+     case "3" -> s.substring(1);
+     default -> s.replace('a', 'b');
+    };
+  }
```

Comment:
We could add a test when the switch uses `yield`.

```java
    String v = switch (cond) {
      case "1": yield "aValue";
      case "2": yield s.toString();
      case "3": {
        if (cond.equals("")) {
          yield s.substring(1);
        } else {
          yield s.substring(2);
        }
      }
      default: yield s.replace('a', 'b');
    };
```

It works as expected, but it's not obvious it does at first glance.